.husky/pre-commit

```diff
@@ -5,7 +5,6 @@
 # This hook is triggered first before even getting commit message.
 #
 
-yarn lint:fix
```

Comment:
laissons le lint

Reply:
the lint takes too much time to find an alternative

Reply:
so comment it